.rubocop.yml

```diff
@@ -17,18 +21,43 @@ Bundler/DuplicatedGem:
 Bundler/OrderedGems:
   Enabled: true
 
+Layout/FirstArrayElementIndentation:
+  EnforcedStyle: consistent
+
+Layout/LineLength:
+  Max: 100
+
+Lint/Debugger:
+  Exclude:
+    - "config/application.rb"
+
+Lint/ScriptPermission:
+  Exclude:
+    - "Rakefile"
+
+Metrics/AbcSize:
+  Max: 35
+  Exclude:
+    - "spec/**/*"
+
 Metrics/BlockLength:
+  Max: 50
   Exclude:
-    - "spec/**/*_spec.rb"
-    - "config/environments/**/*.rb"
+    - "config/**/*"
+    - "spec/**/*"
 
-Metrics/LineLength:
-  Max: 100
+Metrics/ClassLength:
+  Max: 250
```

Comment:
This seems like a lot of lines for a class? The default is already at 100.

Reply:
yea I disagree with this change as well...
if you class has that many lines it might be doing too much 📟